src/pages/security.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import clsx from 'clsx';
+import Layout from '@theme/Layout';
+import Link from '@docusaurus/Link';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+import styles from './styles.module.css';
+
+export default function Security() {
+  const context = useDocusaurusContext();
+  const { siteConfig = {} } = context;
+  return (
+    <Layout
+      title="Temporal Security"
+      permalink="/security"
+      description="<head />"
+    >
+      <header className={clsx('hero hero--primary', styles.heroBanner)}>
+        <div className="container">
+          <h1 className="hero__title">Temporal security</h1>
+          <p className="hero__subtitle">Our commitment</p>
+        </div>
+      </header>
+      <div className={clsx('hero hero--secondary', styles.heroBanner)}>
+        <div className="container">
+          <div className="row">
+            <div className={clsx('col col--6', styles.justifyRight)}>
+              <h2>Our Stack</h2>
+            </div>
+            <div className={clsx('col col--4', styles.justifyLeft)}>
+              <p>The Temporal Cloud, an instance of our open source <a href="https://github.com/temporalio/temporal">Temporal Server software</a>, is hosted on AWS and we
+              use <a href="https://www.datastax.com/products/datastax-astra">Astra Datastax</a> for data storage.</p>
+            </div>
+          </div>
+          <div className="row">
+            <div className={clsx('col col--6', styles.justifyRight)}>
+              <h2>Encryption</h2>
+            </div>
+            <div className={clsx('col col--4', styles.justifyLeft)}>
+              <p>All traffic to and from the Temporal Cloud is encrypted with mTLS.</p>
+              <p>Customers have access to data encryption tools to encrypt data at rest.</p>
+            </div>
```

Comment:
perhaps add a link to further details here https://docs.temporal.io/docs/server/security

i am 10% worried that having two "Security pages" will be confusing if we don't handle the communication right. this one is for compliance folks, the other is for developers looking to lock down temporal.